tests_e2e/test_suites/ext_policy_with_dependencies.yml

```diff
@@ -0,0 +1,11 @@
+#
+# The test suite verifies that disallowed extensions are not processed, but the agent should still report status.
+#
+name: "ExtPolicyWithDependencies"
+tests:
+  - "ext_policy/ext_policy_with_dependencies.py"
+images: "random(endorsed)"
```

Comment:
same comment here, we should get more coverage than 1 run per day, maybe consider running on all endorsed, or 5-10 endorsed images per day

Reply:
Updated to all endorsed, but I can change to 5-10 if this adds too much overhead.